wiki/Beatmapping/Beatmap_submission/es.md

```diff
@@ -0,0 +1,27 @@
+# Envío
+
+Los [beatmaps](/wiki/Beatmap) se pueden enviar al sitio web de osu! a través del [editor dentro del juego](/wiki/Client/Beatmap_editor). Enviar un beatmap te permite recibir atención de otros usuarios y potencialmente ingresar a las categorías [clasificados](/wiki/Beatmap/Category#clasificados) o [amados](/wiki/Beatmap/Category#amados).
+
+![](img/bss_warning-ES.png "Pantalla del sistema de envío de beatmaps")
+
+Al seleccionar `Subir mapa...` del menú desplegable `Archivo` en el editor (atajo: `Ctrl` + `Shift` + `U`) se abrirá la ventana del **sistema de envío de beatmaps** (***BSS***). Inicialmente, esto enumera recursos para ayudar a los usuarios a hacer preguntas relacionadas con el mapping, encontrar [comentarios](/wiki/Modding) para sus beatmaps y garantizar que sus mapas sean adecuados para clasificar. Si tiene problemas al utilizar el BSS, véase la guía [problemas del BSS](/wiki/Guides/BSS_issues).
+
+Si el beatmap que está subiendo un usuario aún no está en el sitio web de osu!, la ventana del BSS indicará cuántas veces más puede publicar un usuario. Si el beatmap está nominado, la ventana advierte que la nominación se restablecerá por actualizar el beatmap. Si el beatmap está en el [cementerio](/wiki/Beatmap/Category#abandonados), la ventana advierte que el beatmap se elevará a la categoría pendiente.
```

Comment:
```suggestion
Si el beatmap que está subiendo un usuario aún no está en el sitio web de osu!, la ventana del BSS indicará cuántas veces más puede publicar un usuario. Si el beatmap está nominado, la ventana advierte que la nominación se restablecerá por actualizar el beatmap. Si el beatmap está en el [cementerio](/wiki/Beatmap/Category#abandonados), la ventana advierte que el beatmap se elevará a la categoría pendientes.
```

Reply:
La categoría es `pendiente`, ¿no? Además está hablando de un solo beatmap

Reply:
y deja de cerrar revisiones

Reply:
Ok, aunque en la página también dice [`pendiente`](https://cdn.discordapp.com/attachments/1129500906195525743/1169355881016733706/lvVyeLr417.png?ex=65551a87&is=6542a587&hm=d64b15410da09ca8871ae25f4386356798105c218676b102eb9e0a570f5dd64a&)